src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,33 @@
+package seedu.addressbook.data.tag;
+
+import java.util.ArrayList;
+
+/*
+ * Association class;
+ * To print out all additions and removal of tags when the programme exits
```

Comment:
A more precise description is `Represents an adding or removing a Tag to/from a Person`.